terminal/display_posix.go

```diff
@@ -0,0 +1,11 @@
+// +build !windows
+
+package terminal
+
+import (
+	"fmt"
+)
+
+func EraseInLine(mode EraseLineMode) {
```

Comment:
Yea sorry, could have been more clear. I do see the benefit of sticking close to the old API but I think i prefer `EraseLine`. The "In" feels very repetitive unnecessarily